### ISSUE TITLE
Tagging does nothing on Windows 10

### ISSUE THREAD

**Issue by drwbns** [Jamminroot/intag#4]
Not sure what is going wrong. I installed it and it shows on the context menu but when I add tags to folder, the tags don't show in explorer.

**Comment by Jamminroot** [comment]
Yup, that was me who forgot to add one line in the recent release (1.2.0.0). I've fixed that in 1.2.1.0, I'll upload that in a minute

**Comment by Jamminroot** [comment]
Reopen please if that's not fixed

**Comment by drwbns** [comment]
It works now. Thanks! Is there any chance in the future that it can support multiple words per tag like a short description?

**Comment by Jamminroot** [comment]
Not sure I understood you tbh, can you elaborate?
Do you mean like a tag consisting of a few words? Or a combination of tags when once added a certain phrase - multiple tags are added? Or adding tag for multiple folders maybe?

For the first case, that should already work:
![image](https://user-images.githubusercontent.com/6387654/142389542-ae74399f-6ce0-48d8-8af4-e0d3c04d6836.png)

**Comment by drwbns** [comment]
I found that it can display long text but the UI wont display strings of
more than about 17 characters. Can the UI be converted to display longer
text descriptions and possibly infotips for very long tags?

**Comment by Jamminroot** [comment]
Ohhh, I see. Yeah, I'll fix that somewhere by the weekend

**Comment by Jamminroot** [comment]
Please have a look at fixed version
Fixed in https://github.com/Jamminroot/intag/releases/tag/1.2.2.0

**Comment by drwbns** [comment]
Yes, the infotips work wonders! Thank you!
